src/routes/index.ts

```diff
@@ -562,7 +562,7 @@ router.route(config.RELEVANT_PERIOD_OWNED_LAND_FILTER_URL)
     companyAuthentication,
     navigation.hasOverseasEntity)
   .get(relevantPeriodOwnedLandFilter.get)
-  .post(relevantPeriodOwnedLandFilter.post);
+  .post(...validator.relevantPeriodOwnedLand, relevantPeriodOwnedLandFilter.post);
```

Comment:
@ConnorOS-ch this could be resolved now as @ndeekonda-ch did this a few days ago in commit [bb8b28d](https://github.com/companieshouse/overseas-entities-web/pull/1405/commits/bb8b28d1def3133a24ed6dd38436e3eacac08a95).

Reply:
yes I have added checkValidations in my commit I am not sure why it is not showing here.